manifests/p/PassmarkSoftware/Fragger/1.0/PassmarkSoftware.Fragger.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: PassmarkSoftware.Fragger
+PackageVersion: 1.0

```

Comment:
A warning to the people who looked up the PR because of the Upgrade loop, the Developer needs to add the DisplayVersion string to the uninstall registry key.